integration-tests/tests/src/tests/sync/asymmetric.ts

```diff
@@ -46,23 +50,41 @@ describe.skipIf(environment.missingServer, "Asymmetric sync", function () {
     });
 
     it("Schema with asymmetric = true and embedded = false", function () {
```

Comment:
This is how other tests ensure the `this.realm` is available for the test function.

```suggestion
    it("Schema with asymmetric = true and embedded = false", function (this: RealmContext) {
```

Reply:
Thanks!